webpack.parts.js

```diff
@@ -62,9 +62,7 @@ exports.noParseD3Vendor = () => ({
 });
 
 exports.externals = () => ({
-    externals: {
-        d3: 'd3',
-    },
+    externals: /^d3-/,
```

Comment:
what does this do mean?

Reply:
This removes all the d3 modules from the bundle!